Compile error with `DEBUG_LOG_VERBOSE` enabled

## Current Behavior

When adding `-DDEBUG_LOG_VERBOSE` to `super_defines.txt`, ELRS won't compile:

```
In file included from lib/CrsfProtocol/CRSFEndpoint.cpp:4:
lib/CrsfProtocol/CRSFEndpoint.cpp: In member function 'void CRSFEndpoint::parameterUpdateReq(crsf_addr_e, bool, uint8_t, uint8_t, void*)':
lib/CrsfProtocol/CRSFEndpoint.cpp:352:40: error: 'fieldId' was not declared in this scope
         DBGVLN("Read parameter %u %u", fieldId, fieldChunk);
                                        ^~~~~~~
lib/logging/logging.h:63:26: note: in definition of macro 'DBGLN'
       debugPrintf(msg, ##__VA_ARGS__); \
                          ^~~~~~~~~~~
lib/CrsfProtocol/CRSFEndpoint.cpp:352:9: note: in expansion of macro 'DBGVLN'
         DBGVLN("Read parameter %u %u", fieldId, fieldChunk);
         ^~~~~~
lib/CrsfProtocol/CRSFEndpoint.cpp:352:40: note: suggested alternative: 'yield'
         DBGVLN("Read parameter %u %u", fieldId, fieldChunk);
                                        ^~~~~~~
lib/logging/logging.h:63:26: note: in definition of macro 'DBGLN'
       debugPrintf(msg, ##__VA_ARGS__); \
                          ^~~~~~~~~~~
lib/CrsfProtocol/CRSFEndpoint.cpp:352:9: note: in expansion of macro 'DBGVLN'
         DBGVLN("Read parameter %u %u", fieldId, fieldChunk);
         ^~~~~~
lib/CrsfProtocol/CRSFEndpoint.cpp:352:49: error: 'fieldChunk' was not declared in this scope
         DBGVLN("Read parameter %u %u", fieldId, fieldChunk);
                                                 ^~~~~~~~~~
lib/logging/logging.h:63:26: note: in definition of macro 'DBGLN'
       debugPrintf(msg, ##__VA_ARGS__); \
                          ^~~~~~~~~~~
lib/CrsfProtocol/CRSFEndpoint.cpp:352:9: note: in expansion of macro 'DBGVLN'
         DBGVLN("Read parameter %u %u", fieldId, fieldChunk);
         ^~~~~~
*** [.pio/build/Unified_ESP32_2400_RX_via_WIFI/lib7dc/CrsfProtocol/CRSFEndpoint.cpp.o] Error 1
```

This is, I think, due to `fieldId` being used at:

https://github.com/ExpressLRS/ExpressLRS/blob/66b84af838694ea15bf39268aeb468b191d477cf/src/lib/CrsfProtocol/CRSFEndpoint.cpp#L352

but it's not defined anywhere.

This line of code seems to be a relic from older code, previously added in `src/lib/LUA/lua.cpp` in dc36257 back in the day, but the code has been refactored since then.

I don't know if `fieldChunk` exists in that scope either.

## Steps to Reproduce

Attempt to compile `master` with `DEBUG_LOG_VERBOSE` defined. With just `DEBUG_LOG` defined, it compiles fine. Only the verbose gives this error.

## Possible Solution (Not obligatory)

`fieldId` probably has a new name, but I don't know which one.

## Details

N/A

## Your Environment

- user_defines: 
```
-DDEBUG_LOG
-DDEBUG_LOG_VERBOSE
```

--- a/src/lib/CrsfProtocol/CRSFEndpoint.cpp
+++ b/src/lib/CrsfProtocol/CRSFEndpoint.cpp
@@ -349,7 +349,7 @@ void CRSFEndpoint::parameterUpdateReq(const crsf_addr_e origin, const bool isElr
         break;
 
     case CRSF_FRAMETYPE_PARAMETER_READ: {
-        DBGVLN("Read parameter %u %u", fieldId, fieldChunk);
+        DBGVLN("Read parameter %u %u", parameterIndex, parameterArg);
         if (parameterIndex < MAX_CRSF_PARAMETERS && parameter)
         {
             const auto field = (commandParameter *)parameter;

